doc/usecases/record_elektra/UC_cmerge_conflict_keys.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Get information which keys are conflicting in 3-way-merge
```

Comment:
Probably too low-level to be needed as UC. It is simply missing functionality, needed for the merge UC.